ui/app/i18n/clinical/locale_en.json

```diff
@@ -197,7 +209,20 @@
     "CLINICAL_DIAGNOSIS_CERTAINTY_PRESUMED": "PRESUMED",
     "CLINICAL_ACCEPT":"Accept",
     "CLINICAL_ACTION":"Action",
+    "CLINICAL_DIAGNOSIS": "Diagnosis",
+    "CLINICAL_ORDER": "Order",
+    "CLINICAL_DIAGNOSIS": "Diagnosis",
```

Comment:
@rbuisson `"CLINICAL_DIAGNOSIS": "Diagnosis"` appears twice.

Reply:
Thanks. Should be good now.